Annotato/Annotato/Views/Components/Extensions/UIViewController.swift

```diff
@@ -19,6 +19,24 @@ extension UIViewController {
         return viewController
     }
 
+    static func instantiateFullScreenFromStoryboard(
+        _ storyboard: Storyboard,
+        documentPdfViewModel: DocumentPdfViewModel
+    ) -> Self? {
+        let viewController = instantiateFromStoryboard(storyboard)
+        viewController?.modalPresentationStyle = .fullScreen
+
+        if let viewController = viewController as? DocumentEditViewController {
+            let currentDocumentViewModel = DocumentViewModel(
+                annotations: SampleData().exampleAnnotations(),
+                pdfDocument: documentPdfViewModel
+            )
+            viewController.currentDocumentViewModel = currentDocumentViewModel
+        }
```

Comment:
Can you move this out? The single responsibility of this function is to present a full screen view controller. If we load various data based on what the view controller is, the function will be flooded with if statements. Isn't it better to place this under `Navigable` which has a `goToDocumentEdit` function specifically for presenting `DocumentEditViewController`?.

```
    func goToDocumentEdit(documentPdfViewModel: DocumentPdfViewModel) {
        guard let viewController = DocumentEditViewController.instantiateFullScreenFromStoryboard(
            .document, documentPdfViewModel: documentPdfViewModel
        ) else {
            return
        }

        // put the data here
        let currentDocumentViewModel = ... // in future this will be a load method by a manager or something
        viewController.currentDocumentViewModel = ...

        present(viewController, animated: true, completion: nil)
    }
```